test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -4905,6 +4905,24 @@ def forward(self, x, y):
         y = torch.randint(10, (2, 3, 5), dtype=torch.long)
         self.run_test(XorModel(), input_args=(x, y))
 
+    @skipIfUnsupportedMinOpsetVersion(9)
```

Comment:
Enable a opinfo test? I am finishing up https://github.com/pytorch/pytorch/pull/92087 which also fixes the test file.

Reply:
Done, very robust this time.